lib/api/objectPutPart.js

```diff
@@ -221,9 +227,14 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                             objectLocationConstraint, cipherBundle, splitter,
                             partInfo);
                     }
-                    // if data backend doesn't handle MPU, continue waterfall
-                    return next(null, destinationBucket,
-                    objectLocationConstraint, cipherBundle, splitter, null);
+                    let msg = 'backend is managing MPU but was';
+                    msg += ' not handle after uploadPart';
```

Comment:
could also use constant but it's okay either way
```suggestion
                    const msg = 'backend is managing MPU but was'
                        + ' not handle after uploadPart';
```